src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Security/index.html.twig

```diff
@@ -0,0 +1,59 @@
+{#**
+  * Copyright since 2007 PrestaShop SA and Contributors
+  * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+  *
+  * NOTICE OF LICENSE
+  *
+  * This source file is subject to the Open Software License (OSL 3.0)
+  * that is bundled with this package in the file LICENSE.md.
+  * It is also available through the world-wide-web at this URL:
+  * https://opensource.org/licenses/OSL-3.0
+  * If you did not receive a copy of the license and are unable to
+  * obtain it through the world-wide-web, please send an email
+  * to license@prestashop.com so we can send you a copy immediately.
+  *
+  * DISCLAIMER
+  *
+  * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+  * versions in the future. If you wish to customize PrestaShop for your
+  * needs please refer to https://devdocs.prestashop.com/ for more information.
+  *
+  * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+  * @copyright Since 2007 PrestaShop SA and Contributors
+  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+  *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Admin.Advparameters.Feature" %}
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{% block content %}
+  <div class="row justify-content-center">
+    {{ form_start(generalForm, {attr : {class: 'form col-xl-10'}, action: path('admin_security_general_save') }) }}
+      {% block administration_form_general %}
+        <div class="card" id="configuration_fieldset_general">
+          <h3 class="card-header">
+            <i class="material-icons">settings</i> {{ 'General'|trans({}, 'Admin.Global') }}
+          </h3>
+          <div class="card-block row">
+            <div class="card-text">
+              {{ form_row(generalForm.token) }}
```

Comment:
Better indeed, but I think you can just do:

```
{{ form_row(generalForm) }}
```

And you need to assign the latest prestahop-ui theme to the form for everything to work fine, because it's not the default one yet.

```
{% form_theme generalForm 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit.html.twig' %}
```

And maybe it's actually `form_widget(generalForm)` instead of `form_row` To be tested

Reply:
I can't only do `{{ form_row(generalForm) }}` the action is different.

And it's `form_row` because it renders the widget
```twig
{%- block form_row -%}
    <tr>
        <td>
            {{- form_label(form) -}}
        </td>
        <td>
            {{- form_errors(form) -}}
            {{- form_widget(form) -}}
        </td>
    </tr>
{%- endblock form_row -%}
```

When`form_widget` does this:
```twig
{%- block form_widget -%}
  {% if compound %}
    {{- block('form_widget_compound') -}}
  {% else %}
    {{- block('form_widget_simple') -}}
  {% endif %}

  {{- block('form_help') -}}
{%- endblock form_widget -%}

```